fms2_io/netcdf_io.F90

```diff
@@ -81,7 +81,9 @@ module netcdf_io_mod
 type, public :: Valid_t
   logical :: has_range !< Flag that's true if both min/max exist for a variable.
   logical :: has_fill !< Flag that's true a user defined fill value.
+  logical :: has_missing !< Flag that's true a user defined missing value.
```

Comment:
Remove one of the `has_missing` declarations

Reply:
Done